zkir/Dialect/Field/IR/FieldTypes.td

```diff
@@ -137,13 +137,15 @@ def Field_QuadraticExtFieldType : Field_Type<"QuadraticExtField", "f2", [
   let summary = "Quadratic extension field element type";
   let description = [{
     Represents an element in a quadratic extension field defined by the given
-    prime field `baseField` and the irreducible binomial `beta`.
+    prime field `baseField` and the irreducible binomial `nonResidue`.
+
+    An element is represented as a₀ + a₁u where u² = ξ.
```

Comment:
```suggestion
    An element is represented as a₀ + a₁v where v² = ξ.
```

I think you use `v` in other extension fields so let's keep it consistent. Also, the `xi` is otherwise not defined in the below equation.